src/ProjectTemplates/Web.ItemTemplates/content/Protobuf/.template.config/template.json

```diff
@@ -13,8 +13,8 @@
     "type": "item"
   },
   "groupIdentity": "Microsoft.Web.Grpc.Protobuf",
-  "precedence": "600",
```

Comment:
Likewise some others were at 600, should these stay there or go to 9800? @MackinnonBuck